testutil/testutil.go

```diff
@@ -41,7 +41,7 @@ var UseSpewToPrettyPrintArrays = true
 var PanicOnTestSetupFailure = true
 
 // Set this flag "true" to print a stack stace when a test failed
-var PanicOnTestFailure = false
+var PanicOnTestFailure = !false
```

Comment:
I guess you used `!` to switch over quickly on your end, but please change back to false.